test/SILOptimizer/address_lowering.sil

```diff
@@ -1170,6 +1171,85 @@ bb2(%9 : $Error):
   throw %9 : $Error
 }
 
+sil [ossa] @use_T : $@convention(thin) <T> (@in T) -> ()
+
+// CHECK-LABEL: sil [ossa] @test_checked_cast_br1 : $@convention(method) <T> (@in Any) -> () {
+// CHECK: bb0(%0 : $*Any):
+// CHECK:   [[FAILURE_DST:%.*]] = alloc_stack $Any
+// CHECK:   [[SUCCESS_DST:%.*]] = alloc_stack $T
+// CHECK:   checked_cast_addr_br take_on_success Any in %0 : $*Any to T in [[SUCCESS_DST]] : $*T, bb2, bb1
+// CHECK: bb1:
```

Comment:
I'm not sure why the SIL printer lays out the false block first (we could consider changing that, possibly with a flag to avoid rewriting old tests). Either way, it makes the tests easier to read if the input and output CFGs are consistent.

Reply:
Looks like that's the order we get from rpo traversal while printing with `-emit-sorted-sil`